java11/adoptopenjdk/11.0.3_7/hotspot/Dockerfile

```diff
@@ -8,8 +8,7 @@ ENV NON_ROOT_CONTAINER_USER bob-the-builder
 ENV DOCKER_COMPOSE_VERSION 1.24.0
 ENV DOCKER_COMPOSE_BASE_DOWNLOAD_URL https://github.com/docker/compose/releases/download
 
-RUN groupadd -g 1000 ${NON_ROOT_CONTAINER_USER} && \
-    useradd -u 1000 ${NON_ROOT_CONTAINER_USER} -g ${NON_ROOT_CONTAINER_USER}
+RUN useradd -rm -d /home/${NON_ROOT_CONTAINER_USER} -s /bin/bash -g root -G sudo -u 1000 ${NON_ROOT_CONTAINER_USER}
```

Comment:
I like your changes the only thing i would like to avoid is to make the NON_ROOT_CONTAINER_USER part of the root group because the idea is to run the build images as non root.

Reply:
I'll merge the pull request and keep the NON_ROOT_CONTAINER_USER as non root.